src/index.ts

```diff
@@ -135,6 +131,7 @@ export type AfterDeleteTriggerFunctionEvent<
   ExtendObjectT = AnyObject
 > = {
   data: DataT & IDBObject;
+  originalData: OriginalDataT;
```

Comment:
@Rusty-Beard as I remember we don't have `originalData` here, right?

Reply:
It is defined in the generic, so I added this line (just in case)